web/styles/message_header.css

```diff
@@ -163,6 +169,10 @@
         &.sticky_header {
             box-shadow: var(--unread-marker-left) 0 0 0 var(--color-background);
 
+            .recipient_bar_control_button {
+                visibility: visible !important;
```

Comment:
Is this necessary?
Things seem to work fine even with this removed.

Reply:
This is to make sure sticky headers have persistent recipient bar controls